spaceship/lib/spaceship/commands_generator.rb

```diff
@@ -39,9 +39,10 @@ def run
       command :spaceauth do |c|
         c.syntax = 'fastlane spaceship spaceauth'
         c.description = 'Authentication helper for spaceship/fastlane to work with Apple 2-Step/2FA'
+        c.option('--exports_to_clipboard', 'Whether the export method should be copied to clipboard. When using this option, the session can be exported to an env var by calling `eval $(pbpaste)`')
```

Comment:
I think I saw somebody else suggest this but we can possibly rename this to `copy_to_clipboard` 😇 “Copy to clipboard” tends to be what the standard phrase is for this. Sorry if that is a lot of work 😔

Reply:
Yup! Np 😊